react/Autosuggest/Autosuggest.js

```diff
@@ -1,24 +1,26 @@
 import styles from './Autosuggest.less';
-
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
 import ReactAutosuggest from 'react-autosuggest';
 import IsolatedScroll from 'react-isolated-scroll';
-
 import omit from 'lodash/omit';
-
 import TextField from '../TextField/TextField';
-
 import smoothScroll from '../private/smoothScroll';
 import smallDeviceOnly from '../private/smallDeviceOnly';
 
+/* eslint-disable react/no-deprecated */
```

Comment:
I had to add this comment here (and in `CustomMonthPicker.js`) because of an eslint-plugin-react update, due to our use of `componentWillUpdate`. To maintain backwards compatibility, I've left it alone for now and ignored the ESLint rule, but we'll have to rename/remove it when we upgrade to React v17.

Reply:
How about marking it as a warning so it's visible?